src/tests/unit/tests/background/stores/visualization-scan-result-store.test.ts

```diff
@@ -289,8 +289,8 @@ describe('VisualizationScanResultStoreTest', () => {
 
         const expectedState = new VisualizationScanResultStoreDataBuilder()
             .withSelectorMap(type, selectorMap)
-            .withFullIdToRuleResultMap(type, expectedFullIdToResultMap)
-            .withSelectedIdToRuleResultMap(type, expectedFullIdToResultMap)
+            .withFullIdToRuleResultMapForIssues(expectedFullIdToResultMap)
```

Comment:
It feels a little off for half of the builder invocation to use `type` and the other half to use hardcoded Issues based things. I think it would be nicer to inline the `type` variable (like the `onUpdateIssuesSelectedTargets` test does on line 395) to make it more apparent that they're hardcoded in sync.

Reply:
The point is we are not passing any other `VisualizationType` except for `Issues` hence the name change.